api/WebGL2RenderingContext.json

```diff
@@ -3127,6 +3127,39 @@
           }
         }
       },
+      "drawingBufferColorSpace": {
+        "__compat": {
+          "spec_url": "https://registry.khronos.org/webgl/specs/latest/1.0/#DOM-WebGLRenderingContext-drawingBufferColorSpace",
```

Comment:
Almost all other spec URLs in this file are to numbered sections. Can you match that?

Reply:
There's actually a number of spec URLs that point to places like this, and I feel that's better because it points to the exact feature. We should probably change the numbered ones accordingly.

Reply:
The numbered sections style is still completely dominant. I'd suggest sticking to that, and then opening a PR to change everything to your preferred style, asking review from whoever added the current spec URLs.

Reply:
I decided to look into this, and I figured out why there's a lot of numbered sections used rather than the exact method or property.  It's because the link to the section is the closest available!  Features like `activeTexture` or `attachShader` don't have named anchors that point directly to them, so the section header has to be used in its place.

Reply:
I see, so it's the exception that there's a direct link. In fact, just 11 cases like this one. `Array.from(document.querySelectorAll('[id]')).map(e => e.id).filter(id => id.startsWith('DOM'))` gives:

```
DOM-WebGLActiveInfo-size
DOM-WebGLActiveInfo-type
DOM-WebGLActiveInfo-name
DOM-WebGLShaderPrecisionFormat-rangeMin
DOM-WebGLShaderPrecisionFormat-rangeMax
DOM-WebGLShaderPrecisionFormat-precision
DOM-WebGLRenderingContext-canvas
DOM-WebGLRenderingContext-drawingBufferWidth
DOM-WebGLRenderingContext-drawingBufferHeight
DOM-WebGLRenderingContext-drawingBufferColorSpace
DOM-WebGLRenderingContext-unpackColorSpace
```

If you want to use these IDs where available, can you submit a PR updating the rest?

Reply:
Just did a check, and all of the other features have the appropriate URLs!

Reply:
Cool! I already approved, I'll merge as well now then.